providers/poe/models/Claude-Sonnet-4.toml

```diff
@@ -0,0 +1,16 @@
+name = "Claude-Sonnet-4"
+release_date = "2025-05-21"
+last_updated = "2025-05-21"
+attachment = false
+reasoning = false
+temperature = true
+tool_call = false
+open_weights = false
+
+[limit]
+context = 128_000
+output = 16_384
+
+[modalities]
+input = ["text"]
+output = ["text"]
```

Comment:
Most models in this PR have:
- `reasoning` and `tool_call` set to `false`
- `context` and `output` limit set to `128000` and `15384`
- `input modalities` set to `text`

They seems to be inaccurate.